src/FSLibrary/MissionHistoryGenerateAndCatchup.fs

```diff
@@ -15,7 +15,7 @@ open StellarSupercluster
 let sorobanProtocolVersion = 20
 
 let historyGenerateAndCatchup (context: MissionContext) =
-    let context = context.WithNominalLoad
+    let context = context.WithNominalLoad.WithSmallLoadgenOptions
```

Comment:
Isn't this a no-op now, since we don't set number of accounts and txs anymore? I think we can keep all the places that use the defaults (withNominalLoad, basic presets for account generation, pay, etc) as-is as to not accidentally change the existing missions.

Reply:
I don't think so? My reading of the documentation for [the `with` keyword for record updates](https://learn.microsoft.com/en-us/dotnet/fsharp/language-reference/copy-and-update-record-expressions) (which is how `WithNominalLoad` and `WithSmallLoadgenOptions` work) is that it only updates the specified fields and leaves the others untouched. So `WithNominalLoad` would return a new context with some fields updated, and pass that to `WithSmallLoadgenOptions` which will update different fields. So the end effect of this is that `context` has both the `WithNominalLoad` AND `WithSmallLoadgenOptions` defaults (there's no overlap between these updates).

I'm I misunderstanding how record update in F# works? I'll see if I can test a simple example in the REPL to double check.

Reply:
I was able to check this with a simple example in the REPL:

```F#
> type Rec = { X : int ; Y : int ; Z : int };;
type Rec =
  { X: int
    Y: int
    Z: int }

> type Rec with
-     member self.UpdateX : Rec = { self with X = 4 }
-     member self.UpdateY : Rec = { self with Y = 5 };;
type Rec with
  member UpdateX : Rec
type Rec with
  member UpdateY : Rec

> let r = { Rec.X = 1 ; Rec.Y = 2 ; Rec.Z = 3 };;
val r : Rec = { X = 1
                Y = 2
                Z = 3 }

> r.UpdateX.UpdateY
- ;;
val it : Rec = { X = 4
                 Y = 5
                 Z = 3 }
```

So `context` should have both sets of updates, not just `WithSmallLoadgenOptions`.